README.md

```diff
@@ -1,6 +1,9 @@
 Contao Open Source CMS
 ======================
 
+**This repository holds the source code for version 3.5 of Contao and doesn’t receive bug fixes anymore.**  
+**Please take a look at [contao/core-bundle][10] instead.**
```

Comment:
I don't like this, because it is easy to overlook. We should replace the existing content instead and provide more information like in the updated CONTRIBUTORS.md file. WDYT?

Reply:
Yes, makes sense :)

Reply:
Will do.